python/tests/sql_aggregates/test_min.py

```diff
@@ -0,0 +1,146 @@
+import unittest
+from feldera import PipelineBuilder, Pipeline
+from tests import TEST_CLIENT
+
+class TestAggregatesBase(unittest.TestCase):
+    def setUp(self) -> None:
```

Comment:
min works with almost every data type, including strings.
so these would have to be tested too